python/py-graveyard/Portfile

```diff
@@ -473,6 +473,7 @@ py-webob                1.8.2_1     26
 py-werkzeug             0.14.1_1    26 33
 py-wordaxe              0.3.3_1     26
 py-w3lib                1.9.0_1     33
+py-xarray               0.10.3_1    34
```

Comment:
Since you are updating the version of py-xarray at the same time, you can use that as the version in the graveyard. So change it to:

`py-xarray               0.11.0    34`

Reply:
Oops, I think there should be a consensus - https://github.com/macports/macports-ports/pull/3027#discussion_r234178503

Reply:
Weird, I don't see that comment and I cannot read the full text to see what you said.

Reply:
Maybe that's because I marked that conversation as resolved. The comment should work as usual now.

Reply:
I see it now, thank you. I respectfully disagree. It has been done many times before that way including by the creator of the py-graveyard port.

Reply:
Thanks for explanation. @fhanzer could you change it back? Sorry for inconvenience.

Reply:
Ok, no problem, I changed it back!